Libraries/NativeComponent/PlatformBaseViewConfig.js

```diff
@@ -0,0 +1,507 @@
+          borderEndWidth: true,
+          borderEndColor: {process: require('../StyleSheet/processColor')},
+
+          borderTopLeftRadius: true,
+          borderTopRightRadius: true,
+          borderTopStartRadius: true,
+          borderTopEndRadius: true,
+          borderBottomLeftRadius: true,
+          borderBottomRightRadius: true,
+          borderBottomStartRadius: true,
+          borderBottomEndRadius: true,
+          display: true,
+          zIndex: true,
+
+          // ShadowView properties
+          top: true,
+          right: true,
+          start: true,
+          end: true,
+          bottom: true,
+          left: true,
+
+          width: true,
+          height: true,
+
+          minWidth: true,
+          maxWidth: true,
+          minHeight: true,
+          maxHeight: true,
+
+          // Also declared as ViewProps
+          // borderTopWidth: true,
+          // borderRightWidth: true,
+          // borderBottomWidth: true,
+          // borderLeftWidth: true,
+          // borderStartWidth: true,
+          // borderEndWidth: true,
+          // borderWidth: true,
+
+          marginTop: true,
+          marginRight: true,
+          marginBottom: true,
+          marginLeft: true,
+          marginStart: true,
+          marginEnd: true,
+          marginVertical: true,
+          marginHorizontal: true,
+          margin: true,
+
+          paddingTop: true,
+          paddingRight: true,
+          paddingBottom: true,
+          paddingLeft: true,
+          paddingStart: true,
+          paddingEnd: true,
+          paddingVertical: true,
+          paddingHorizontal: true,
+          padding: true,
+
+          flex: true,
+          flexGrow: true,
+          flexShrink: true,
+          flexBasis: true,
+          flexDirection: true,
+          flexWrap: true,
+          justifyContent: true,
+          alignItems: true,
+          alignSelf: true,
+          alignContent: true,
+          position: true,
+          aspectRatio: true,
+
+          // Also declared as ViewProps
+          // overflow: true,
+          // display: true,
+
+          direction: true,
+
+          style: ReactNativeStyleAttributes,
+
+          // [macOS 
+          ...(Platform.OS === 'macos' && {
+            acceptsFirstMouse: true,
+            accessibilityTraits: true,
+            cursor: true,
+            draggedTypes: true,
+            enableFocusRing: true,
+            nextKeyViewTag: true,
+            onBlur: true,
+            onClick: true,
+            onDoubleClick: true,
+            onDragEnter: true,
+            onDragLeave: true,
+            onDrop: true,
+            onFocus: true,
+            onKeyDown: true,
+            onKeyUp: true,
+            onMouseEnter: true,
+            onMouseLeave: true,
+            tooltip: true,
+            validKeysDown: true,
+            validKeysUp: true,
+            nextKeyViewTag: true
```

Comment:
Speaking of this.. anyone against undoing https://github.com/microsoft/react-native-macos/pull/769 ? It's a prop that I added that:
- Won't work in Fabric (relies on deprecated API)
- Requires very specific changes to your native window (autoRecalculatesKeyViewLoop = false)
- Never used by client teams because we just created a FocusZone.

Reply:
Made #1621 . Resolving this comment.